src/main/java/digital/slovensko/autogram/ui/gui/GUIValidationUtils.java

```diff
@@ -43,11 +43,9 @@ public static GridPane createSignatureTableRows(Reports reports, boolean isValid
         var headerLink = new TextFlow(createSignatureTableLink(callback));
         table.addRow(0, headerText, headerLink);
 
-        var leftConstraints = new ColumnConstraints();
-        leftConstraints.prefWidthProperty().bind(table.widthProperty().multiply(0.5));
-        var rightConstraints = new ColumnConstraints();
-        rightConstraints.prefWidthProperty().bind(table.widthProperty().multiply(0.5));
-        table.getColumnConstraints().addAll(leftConstraints, rightConstraints);
```

Comment:
Cele som to musel kompletne rozobrat a zistilo sa, ze toto zabija windows. proste ani za svet s tymto nefunguje ten sizeToScene potom.